test/testcommon/src/main/java/io/pravega/test/common/TestUtils.java

```diff
@@ -85,6 +86,34 @@ public static void await(Supplier<Boolean> condition, int checkFrequencyMillis,
         }
     }
 
+    @SneakyThrows(InterruptedException.class)
```

Comment:
Can you add Java doc for this method for more clarity?

Reply:
done